R/utils.R

```diff
@@ -182,5 +182,6 @@ single_quote <- function(x) {
 }
 
 unlock_environment <- function(x) {
-  .Call(unlock_environment_, x)
+  call <- .Call # confuse R CMD check
+  call("rlang_env_unlock", x, PACKAGE = "rlang")
```

Comment:
Not sure if this is better, but an alternative way around it is

```r
get(".Call")("rlang_env_unlock", x, PACKAGE = "rlang")
```

Reply:
Yeah, that's slightly nicer